src/_guides/language/effective-dart/usage.md

```diff
@@ -1126,9 +1123,7 @@ expression inside:
 may support non-const default values.)
 
 Basically, any place where it would be an error to write `new` instead of
-`const`, Dart 2 allows you to omit the `const`. If your code does not need to
-run on older versions of Dart, you should take advantage of that and drop the
-`const`.
+`const`, Dart 2 allows you to omit the `const`.
```

Comment:
@wmleler @filiph @kevmoo do the tools do a good job with `const` yet?

Reply:
🤷‍♂️